src/modules/mc_hover_thrust_estimator/MulticopterHoverThrustEstimator.cpp

```diff
@@ -113,56 +143,38 @@ void MulticopterHoverThrustEstimator::Run()
 
 	perf_begin(_cycle_perf);
 
-	if (_vehicle_land_detected_sub.updated()) {
-		vehicle_land_detected_s vehicle_land_detected;
-
-		if (_vehicle_land_detected_sub.copy(&vehicle_land_detected)) {
-			_landed = vehicle_land_detected.landed;
-		}
-	}
-
 	if (_vehicle_status_sub.updated()) {
-		vehicle_status_s vehicle_status;
+		vehicle_status_s vehicle_status{};
```

Comment:
Not that important, but you can skip this initialization because the data is only used if the copy succeeds.